ideascube/serveradmin/catalog.py

```diff
@@ -60,6 +60,111 @@ def to_file(self, path):
             f.write(yaml.safe_dump(d, default_flow_style=False))
 
 
+class Handler:
```

Comment:
Moved before Package class to be able to reference handlers as class properties.